example/src/test-data.ts

```diff
@@ -1,4 +1,5 @@
-import { Node, WaterfallIntervals } from '../../src';
```

Comment:
Please revert changes to this file. You have broken the generation of the flame chart data. You can only add a function that generates data for your plugin.

Reply:
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/4976306/211203018-65aecb18-3d89-493a-ae3d-a25ca51d5dde.png">

Reply:
I didn't spot that - merge conflict from hell with that - sorry.

Reply:
reverted